danieldupriest/reimbursinator
# Create custom user

There is now a CustomUser class being used for reimbursinator users. We can add fields to it in the future if needed. I also added a "firstuser" and a few empty Reports belonging to them for testing purposes.

To test:
1. In the `back/` directory run `pipenv install` to set up a local environment.
2. Run `pipenv shell` to start a shell.
3. Start the backend server with `python3 manage.py runserver` (or maybe `py` or `python` depending)
4. Go to `http://localhost:8000/admin/` and log in with username `admin` and password `admin`.
5. Check out the Users and Reports pages to see the example user and their report objects. All fields/sections are still empty.
